packages/core/theme/src/components/select.ts

```diff
@@ -29,6 +29,23 @@ const select = tv({
     listboxWrapper: "scroll-py-6 max-h-64 w-full",
     listbox: "",
     popoverContent: "w-full p-1 overflow-hidden",
+    clearButton: [
+      "w-4",
+      "h-4",
+      "z-10",
+      "mb-4",
+      "relative",
+      "start-auto",
```

Comment:
Could you align the [input's clear button](https://nextui.org/docs/components/input#clear-button)?
It looks like there's no hover style applied.

Reply:
@abhinav700 is it handled?

Reply:
Yup, hover style is now fixed for select

Regarding input component, isn't it already aligned correctly

![image](https://github.com/user-attachments/assets/d28ce539-1fe6-4731-8630-a39738b28d78)